src/mainboard/protectli/vault_adl_p/hda_verb.c

```diff
@@ -0,0 +1,28 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <device/azalia_device.h>
+
+const u32 cim_verb_data[] = {
+	/* Alderlake-P HDMI */
+	0x8086281c, /* Vendor ID */
+	0x80860101, /* Subsystem ID */
+	7, /* Number of entries */
+	AZALIA_SUBVENDOR(2, 0x80860101),
+	0x00278111,
+	0x00278111,
+	0x00278111,
+	0x00278111,
+	AZALIA_PIN_CFG(2, 0x05, 0x18560010),
+	AZALIA_PIN_CFG(2, 0x06, 0x18560020),
+	AZALIA_PIN_CFG(2, 0x07, 0x18560030),
+	AZALIA_PIN_CFG(2, 0x08, 0x18560040),
+	0x00278100,
+	0x00278100,
+	0x00278100,
+	0x00278100
+};
+
```

Comment:
One newline can be probably dropped?

Reply:
https://github.com/Dasharo/coreboot/pull/455/commits/aafb2783e1929ecea0a321a2503668102a2b00f2#diff-e076132410144ae221c74f79411e0a34c571d629818a6b89e11ff02829eeb30cR24